certified-connectors/Strategic Portfolio Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7505 @@
+        },
+        "FinancialDimensionId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialNodeId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "AllocationCenterId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "TimePeriodName": {
+          "type": "string"
+        },
+        "Value": {
+          "format": "double",
+          "type": "number"
+        },
+        "URL": {
+          "type": "string"
+        },
+        "FinancialEntityId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        }
+      }
+    },
+    "BreakdownValueCreationInformation": {
+      "type": "object",
+      "properties": {
+        "FinancialBreakdownId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialTypeId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialDimensionId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialNodeId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "AllocationCenterId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "TimePeriodName": {
+          "type": "string"
+        },
+        "Value": {
+          "format": "double",
+          "type": "number"
+        },
+        "URL": {
+          "type": "string"
+        },
+        "FinancialEntityId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        }
+      }
+    },
+    "CustomFieldValueCreationInformation": {
+      "type": "object",
+      "properties": {
+        "FinancialTypeId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialDimensionId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "FinancialNodeId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "AllocationCenterId": {
+          "format": "uuid",
+          "type": "string",
+          "example": "00000000-0000-0000-0000-000000000000"
+        },
+        "Value": {},
```

Comment:
@sanda-cheudan can you please explain this empty Value? can you please define or remove?